torch/_inductor/select_algorithm.py

```diff
@@ -1460,6 +1483,17 @@ def get_inputs():
                 )
                 for input_node in input_nodes
             ]
+            
+            # Make sure that all workspace sizes for each choice are the same
+            needs_workspace = any(choice.workspace_arg is not None for choice in choices)
+            if needs_workspace:
+                # TODO right now we only support the same workspace arg for all choices
+                workspace: WorkspaceArg = choices[0].workspace_arg
+                assert all(choice.workspace_arg == workspace for choice in choices)
+                # add 
+                size, zero_fill = workspace.nbytes, workspace.zero_fill
+                workspace_tensor = torch.empty_strided((size,), (1,), dtype=torch.uint8, device="cuda")
```

Comment:
@eellison this part I know is not working correctly/ am not totally sure whats the best way tot allocate this buffer during autotuning